ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
```

Comment:
what's the benefit of using KOMA-Script? (maybe start an issue if this is going to be a lengthy discussion.)

Reply:
nvm i happened to see[this](https://texdoc.org/serve/l2tabuen/0) today.
![image](https://user-images.githubusercontent.com/10174018/108930201-5566b400-75fa-11eb-911d-632873780507.png)

haven't used A4 paper before. TIL.